core/common/src/annotation/TextAnnotation.ts

```diff
@@ -146,11 +146,11 @@ export class TextAnnotation {
    * The anchor point is computed as specified by this annotation's [[anchor]] setting. For example, if the text block is anchored
    * at the bottom left, then the transform will be relative to the bottom-left corner of `textBlockExtents`.
    * The text block will be rotated around the fixed anchor point according to [[orientation]], then the anchor point will be translated by [[offset]].
-   * @param textBlockDimensions The width and height of the bounding box containing the text block. You can compute this using [computeTextBlockExtents]($backend).
+   * @param boundingBox A box fully containing the [[textBlock]]. You can compute this using [computeTextBlockBoundingBox]($backend).
```

Comment:
Does `computeTextBlockBoundingBox` exist? Here and below in `computeTransform`, the jsdoc comment links are not being resolved.

Reply:
No, it doesn't. I thought I was going to have to replace `computeTextBlockExtents` with it but nobody ever used that function.

Reply:
Removed. We can add a link to the new `computeTextBlockLayoutResult` function being added in #6820 after that merges. The existing `layoutTextBlock` function is internal.